modules/@apostrophecms/widget-type/ui/apos/components/AposWidgetEditor.vue

```diff
@@ -218,9 +222,3 @@ export default {
   }
 };
 </script>
-
-<style lang="scss" scoped>
-  .apos-widget-editor ::v-deep .apos-modal__inner {
-    max-width: 458px;
-  }
-</style>
```

Comment:
remove this rule in favor of the one from `AposModal` that already sets the max-width to `540px` as required in the ticket.